apps/scan/frontend/src/screens/poll_worker_screen.tsx

```diff
@@ -246,30 +178,42 @@ export function PollWorkerScreen({
         return;
       }
 
-      const timePollsTransitioned = Date.now();
-      setCurrentPollsTransition(pollsTransition);
+      const pollsTransitionTime = Date.now();
+      await transitionPollsMutation.mutateAsync({
+        type: pollsTransitionType,
+        time: pollsTransitionTime,
+      });
```

Comment:
I'm moving this mutation up in the order of events. Previoiusly it was:

1. Print Report
2. Export CVRs
3. Change Polls State

Part of the reasoning for this was, I believe, that printing the report is so essential that we didn't want to change state without guaranteeing we could do it. But with the ability to reprint reports and the existing ability to re-export CVRs, it's lower risk and I think this is a better order of operations. It makes more sense - for example, the last batch in the CVR export now has an end timestamp, as it should, where previously it may have appeared in the file like it had not ended. cc @arsalansufi I know I've brought up that last bit before.